### PR TITLE
Careers: From Django you were born; to Django you shall return

### PR DESCRIPTION
**Pull request by rosskarchner**
Between the move to Sheer(like) and details missed during the transition, jobs/careers was in a bad state:

- We had never coded for the nuance of jobs overriding the salary range associated with their grade (I spent time looking at the processor and the API code-- I couldn't figure out where this logic was/should go)
- The WYSIWYG editor for job descriptions was gone
- We never came up with a reasonable way for non-devs to trigger 'sheer_index'

This pull request deals with these issues by importing a modified version of the old 'jobmanager' code into our codebase, using our new templates (with some tweaks for dealing with Django objects instead of elasticsearch results). Along the way, I added a static file that django-tinymce needs to function, and fixed the logic around salaries.

## Additions

- listed above

## Removals

- the 'careers' entry has been removed from SHEER_PROCESSORS

## Testing

- Delete the careers data from elasticsearch, you won't need that where we're going: `curl -XDELETE  http://localhost:9200/content/careers`
- Download and unzip [jobs.sql.zip](https://github.com/cfpb/cfgov-refresh/files/236773/jobs.sql.zip), and use you favorite method of loading an SQL file into your database (I tend to `cfgov/manage.py dbshell` and then `source path/to/jobs.sql`)
- make sure the old version of jobmanager isn't creeping in your virtualenv, `pip freeze | grep jobmanager` to see, `pip uninstall jobmanager` to get rid of it.
- -clone private repo CFPB/django-cfgov-common
- `pip install -e ../django-cfgov-common`
- `cfgov/manager.py runserver` and visit http://localhost:8000/about-us/careers and see that everything works the way you expect. Salaries should generally NOT match what's on www, as the grade salaries have been overridden in the Jobs data.
- Log in to http://localhost:8000/django-admin and look at the Jobs there. Observe that the wysisyg controls work in Job descriptions, and that you can edit the salary range min and max, and see the change at http://localhost:8000/about-us/careers/current-openings/
- Observe that you can blank out the salary range min and max, and see the page fall back to the min and max defined for the grade

## Review

- @richaagarwal @kave @kurtw 

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by rosskarchner** [comment]
I did just notice that this breaks tox

**Comment by Scotchester** [comment]
A+++ PR titling

**Comment by rosskarchner** [comment]
Mostly a heads-up for @kave: when this PR is merged, tinymce and reversion will need removed from the deployment base.txt files

**Comment by rosskarchner** [comment]
Travis failure is unrelated

**Comment by rosskarchner** [comment]
@schaferjh this will fix the most urgent careers problems

**Comment by schaferjh** [comment]
thank you @rosskarchner!

File: cfgov/templates/jobmanager/technology-innovation-fellows.html [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (kurtw) [inline cfgov/templates/jobmanager/technology-innovation-fellows.html]: `del whitespace`

**Comment by kurtw** [comment]
👍

**Comment by rosskarchner** [comment]
fixed the whitespace

**Pull request merged**